src/serverlist.cpp

```diff
@@ -575,15 +578,23 @@ void CServerListManager::SlaveServerRegisterServer ( const bool bIsRegister )
         else
         {
             // unregister server
-            eSvrRegStatus = SRS_UNREGISTERED;
-            emit SvrRegStatusChanged();
+            SetSvrRegStatus ( SRS_UNREGISTERED );
 
             pConnLessProtocol->CreateCLUnregisterServerMes ( SlaveCurCentServerHostAddress );
         }
     }
     else
     {
-        eSvrRegStatus = SRS_BAD_ADDRESS;
+        SetSvrRegStatus ( SRS_BAD_ADDRESS );
         emit SvrRegStatusChanged();
     }
 }
+void CServerListManager::SetSvrRegStatus ( ESvrRegStatus eNSvrRegStatus )
+{
+    if ( bConsoleLogging )
```

Comment:
I changed my mind (sorry for that). We should output all registration infos on the console. It does not hurt and my give some users additional valuable information. So please remove all this bConsoleLogging.

Reply:
No problem - saves passing it down :)